nservicebus/azure-service-bus/message-lock-renewal.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Azure Service Bus Transport Message lock renewal
+summary: Extending message lock renewal
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports
+reviewed: 2016-04-20
+---
+
+When messages are received using `PeekLock` mode, the receive operation becomes a two-stage operation. The first stage, the message is locked by the broker for a specific consumer for a fixed period known as `LockDuration`. When the consumer is done with the message, the message is marked as completed by the consumer, indicating to the broker that second stage is finished. Default lock duration for the Azure Service Bus transport is set to 30 seconds. Maximum lock duration allowed by the broker service is 5 minutes. `LockDuration` is a global setting and its value applied to all entities.
```

Comment:
what is the scope of the LockDuration setting? global per appdomain? global per endpoint?